odkx-src/notify-install.rst

```diff
@@ -0,0 +1,79 @@
+Installing ODK-X Notify
+=======================
+
+.. _notify-prereqs:
+
+Prerequisites
+-------------------
+
+- :doc:`sync-endpoint`
+- :ref:`ODK-X Services<services-using>`
+- :ref:`ODK-X Tables<tables-using>`
+- `Java 8 or higher <https://java.com/en/download/>`_
+
+.. note::
+
+    Users who need to use only Skunkworks-Bat app for receiving notifications, for them only :ref:`ODK-X Services<services-using>` is a prerequisite.
+
+.. _notify-setup:
+
+Setting up ODK-X Notify
+-------------------------
+
+:program:`Skunkworks-Parrot (Desktop App)`:
+"""""""""""""""""""""""""""""""""""""""""""
+
+  1. Navigate to https://github.com/odk-x/skunkworks-parrot/releases and download the latest ODK-X_Notify_Desktop.jar file.
+  2. Double click the file to start. If that fails try running:
+
+    .. code-block:: console
+
+  	  $ java -jar path/to/jar
+
+  3. When the application opens, Click Configure. A window shown below will appear
+
+    .. image:: /img/notify-setup/notify-setup.*
+
+  4. `Here <https://drive.google.com/file/d/1OBs5mITcIMREp_q7qKwEwO4XRQjzNnWj/view>`_ you can find step by step instructions for creating a firebase project and adding information in the configure window.
```

Comment:
What is creating a firebase project and why do we need that information?

Reply:
Creating a firebase project is necessary to use the Desktop app. Now, I rephrased the above sentence for more clarity.